namespaces/ci_job.yml

```diff
@@ -16,12 +16,12 @@ namespace:
     description: >
       Name of the CI job.
 
-  - name: build_id
+  - name: number
     type: integer
     description: >
       Build ID of the CI job.
 
-  - name: build_timestamp
+  - name: timestamp
```

Comment:
`@timestamp` may refer to a more precise timing of the specific record. For example xunit test result might have a timestamp, in that case test's timestamp will get elevated to the top level `@timestamp`, while `ci_job.timestamp` will be preserved for the time of Jenkins build.